collections/Auct_E/MS_Auct_E_2_19.xml

```diff
@@ -34,32 +34,68 @@
                      <idno type="SCN">3016</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>
+                  <hi rend="smallcaps">Book of Job with catena compilation</hi>Incomplete catena compiled, possibly by the scribe, from the types known as Γ and w, with material taken from a hitherto unidentified source.</head>
```

Comment:
there might need to be a space before "Incomplete" for this to display correctly